repository/service_request_repository_postgres.go

```diff
@@ -0,0 +1,45 @@
+package repository
+
+import (
+	"clamp-core/models"
+
+	"github.com/google/uuid"
+	"github.com/prometheus/common/log"
+)
+
+type servicerequestrepositorypostgres struct {
```

Comment:
you can move postgres specific implementation to under `postgres` subpackage and name the struct as `ServiceRequestRepository`. So, in the above `init` function we will have `serviceRequestRepository = postgres.ServiceRequestRepository`